server/models/Review.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require("mongoose");
+
+const reviewSchema = new mongoose.Schema({
+  sitterId: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: "User",
+  },
+  clientId: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: "User",
+  },
+  rating: {
+    type: Number,
+    required: true,
```

Comment:
```suggestion
    required: true,
    min: 1,
    max: 5, 
```
Optional: We can add min and max validators here